### DISCUSSION TITLE
[CIP-0019] JWT signer for DID-JWT

### DISCUSSION THREAD

**Discussion by daoauth** [celo-org/celo-proposals#328] [CIP Discussions]
## Overview
- Now signer interface has 4 methods (signTransaction, signPersonalMessage, getNativeKey, decrypt).
- And add a one more signer for did-jwt.
- Because of security, jwt signer and tx signer are using same pk. so it need store in same wallet.
- And it's not possible with ledger-wallet
- It's maybe very useful and strong interface for DIDs. VCs, and more. (eg. HTTP Authentication header type)
```
Authorization: Bearer eyJhbGciOiJIUzI1NiIXVCJ9TJV...r7E20RMHrHDcEfxjoYZgeFONFh7HgQ
```

**Comment by prestwich** [comment]
Associated PR is here:
https://github.com/celo-org/celo-proposals/pull/43
